src/cdk-experimental/testing/protractor.ts

```diff
@@ -0,0 +1,204 @@
+/**
+ * @license
+ * Copyright Google LLC All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+// TODO(mmalerba): Should this file be part of `@angular/cdk-experimental/testing` or a separate
+//  package? It depends on protractor which we don't want to put in the deps for cdk-experimental.
```

Comment:
Discussed this offline and decided it's fine. We won't mention protractor / selenium-webdriver in the package.json, since of course they're dependencies for an e2e test